custom_components/waste_collection_schedule/config_flow.py

```diff
@@ -298,13 +298,21 @@ class WasteCollectionConfigFlow(ConfigFlow, domain=DOMAIN):  # type: ignore[call
     _country: str | None = None
     _source: str | None = None
 
+    _options: dict = {}
     _sources: dict[str, list[SourceDict]] = {}
     _error_suggestions: dict[str, list[Any]]
 
-    def __init__(self, *args: list, **kwargs: dict):
-        super().__init__(*args, **kwargs)
```

Comment:
Removing the constructor with the super call here is probably a bad idea and was probably an accident?

Reply:
No it was not, if a base class has a explicit implementation then it would get called, or? But happy to add it again. (Checked all the base classes and they have not an explicit `__init__`

Reply:
Ah, you're right, missed that.